example/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ['module:metro-react-native-babel-preset'],
```

Comment:
how does the example app know to use the source code for the library without a babel alias?

https://github.com/callstack/react-native-builder-bob/blob/9241f934267a7bfdb2e5233be547eef7c451c4d8/packages/create-react-native-library/templates/example-legacy/example/babel.config.js#L12

Reply:
we don't need a extra babel for it, yarn is enough  
```json
  "dependencies": {
    "@react-native-community/image-editor": "file:..", 
  }
```

Reply:
@retyui it's not quite the same, this depends on the `react-native` key in `package.json` which doesn't need to be source code, and there are plans to remove it in future as well. and the `react-native` key only works for metro, so if you have tests etc. in the example app, it won't import the source code since jest will look for the `main` field and not `react-native` field.

Reply:
I don't really care about `react-native` & `main` differences

yarn  just copy `..` folder to `example/node_modules` as `@react-native-community/image-editor`  package 
and it works as it will be installed thought `npm`

Reply:
> I don't really care about react-native & main differences

you should if you're maintaining a library for react native

> yarn just copy .. folder 

so every time you change something you need to reinstall the deps? seems like a bad dx. there's also a bug in yarn 1 that could result in stale content unless you change the version every time https://github.com/yarnpkg/yarn/issues/6811

you can use `link:` instead but it'll probably result in duplicated `node_modules` that needs to be excluded.

Reply:
>you should if you're maintaining a library for react native

I meant all differences  that you described ins't a problem when I use `file:/path/to/module` as it will work the same as if you install pkg from `npm` registry

>so every time you change something you need to reinstall the deps?

exactly, you need re-install because this library mostly native code only (no js logic) babel  aliases to src won't helps much

> you can use link: instead but it'll probably result in duplicated node_modules that needs to be excluded.

as you know it won't work without  `unstable_enableSymlinks` that ins't production ready yet

Reply:
I pushed commit https://github.com/callstack/react-native-image-editor/pull/124/commits/ef34be6c75bb3d5ae858131e95d592dd24be5d0c that use combination of `yarn link` & `metro.config.js` to resolve alias to local pkg